libs/core/src/components/datepicker/datepicker.trans.styles.scss

```diff
@@ -37,24 +37,24 @@
     background-color: var(--sg-form-control-bg);
     box-sizing: border-box;
     color: var(--sg-text-primary);
+    cursor: text;
     display: flex;
     justify-content: space-between;
-    align-items: stretch;
```

Comment:
This breaks the focus outline:
<img width="219" alt="Screenshot 2024-03-28 at 08 29 10" src="https://github.com/sebgroup/green/assets/428824/feb94e01-b591-484b-9952-eb392138b06a">